backend/internal/adapters/primary/handlers/internal/gemservice.go

```diff
@@ -43,6 +44,32 @@ func (s *gemService) Get(c context.Context, r *gemsapi.GetRequest) (*gemsapi.Get
 	}, nil
 }
 
+func (s *gemService) GetPendingCollectionByUser(c context.Context, r *gemsapi.GetPendingCollectionByUserRequest) (*gemsapi.GetPendingCollectionByUserResponse, error) {
```

Comment:
Also, does the lint check for line length?

Reply:
Hmm, it should! If you run `golangci-lint linters` in the root directory of the backend, `lll` is listed as being enabled. By default, the line length should be set to 120 as seen [here](https://golangci-lint.run/usage/configuration/#config-file). 
I'm not sure why the CI job isn't flagging this. If you run `make lint` locally, does it complain about this? If it does, I probably need to relook the CI job to see if it's working correctly.

Reply:
Hmm, this is the only message I get:

```
Linting source files...
WARN [runner] Can't run linter goanalysis_metalinter: buildir: failed to load package : could not load export data: no export data for "github.com/hansel-app/hansel/protobuf/gemsapi" 
```

Reply:
Weird, I get:
```
Linting source files...
internal/adapters/primary/handlers/internal/gemservice.go:6: File is not `gci`-ed with -local github.com/hansel-app/hansel (gci)

internal/adapters/primary/handlers/internal/gemservice.go:9: File is not `gci`-ed with -local github.com/hansel-app/hansel (gci)
	"google.golang.org/protobuf/types/known/timestamppb"
internal/adapters/primary/handlers/internal/gemservice.go:47: line is 167 characters (lll)
func (s *gemService) GetPendingCollectionByUser(c context.Context, r *gemsapi.GetPendingCollectionByUserRequest) (*gemsapi.GetPendingCollectionByUserResponse, error) {
make: *** [Makefile:43: lint] Error 1
```

Anyway, it seems that the CI job isn't automatically reading from the linter configuration file. I'll take a look into it later.